content/post/know-your-aws-community-day-india-2020-speaker-mayank-raj.md

```diff
@@ -0,0 +1,19 @@
++++
+author = "Dhruv Aggarwal"
+date = 2020-09-23T15:36:47Z
+description = "Mayank Raj will be speaking on \"Serverless Cron with Second level Schedule and Conditional flow\" at <time> on <date> at AWS Community Day India 2020."
+feature_image = "/images/mayank.PNG"
+tags = ["AWS", "AWSCommunityDay2020", "Speaker"]
+title = "Know Your AWS Community Day India 2020 Speaker - Mayank Raj"
+
++++
+Mayank Raj is working as a Solution Architect at Cactus Labs, India will be speaking on "**Serverless Cron with Second level Schedule and Conditional flow**" from 02:00 PM - 02:45 PM Track 2 on 16th October 2020 at AWS Community Day India 2020.
```

Comment:
>  working as a Solution Architect at Cactus Labs, India will be speaking on 


is not right. This should be 

`<name>` is currently `<title>` at `<company>`. `<First name>` will be speaking on <session title>....

Reply:
Corrected.